gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfig.java

```diff
@@ -71,34 +72,4 @@ public LogFilter(String pattern, Integer headerBytes, Integer messageBytes) {
       this.messageBytes = messageBytes;
     }
   }
-
```

Comment:
I notice line 47 `getSampler()` never had a Javadoc comment. Could you add it e.g. used when cloudTracing is enabled and sets the traceConfig's sampler.

Reply:
Done